cms/middleware/toolbar.py

```diff
@@ -47,13 +47,34 @@ def toolbar_plugin_processor(instance, placeholder, rendered_content, original_c
     return output
 
 
+def get_client_ip(request):
```

Comment:
I suggest to implement something like https://github.com/Gidsy/django-geoip-utils/blob/master/geoip_utils/utils.py because there could be other headers that point to the IP and so is best to give developers the option.